conf/machine/skipjack.conf

```diff
@@ -2,14 +2,14 @@
 #@NAME: Ticwatch C2+
 #@DESCRIPTION: Machine configuration for the Ticwatch C2+
 
-require conf/machine/include/tune-cortexa7.inc
+require conf/machine/include/arm/armv7a/tune-cortexa7.inc
 require conf/machine/include/hybris-watch.inc
 
 MACHINE_DISPLAY_ROUND = "true"
 MACHINE_HAS_WLAN = "true"
 
-PREFERRED_VERSION_android = "pie"
+PREFERRED_VERSION_android = "oreo"
 PREFERRED_PROVIDER_virtual/kernel = "linux-skipjack"
 PREFERRED_VERSION_linux = "3.10+pie"
 
-IMAGE_INSTALL += "android-tools android-system underclock msm-fb-refresher asteroid-compass"
+IMAGE_INSTALL += "android-tools android-system underclock android-system-partition msm-fb-refresher asteroid-compass asteroid-2048"
```

Comment:
While I like that you enabled `asteroid-2048` by default, this package shouldn't be installed by default on the stock images.

Maybe instead we should consider adding this game to all watches, but definitely not exclusively this one.

Reply:
Already had a discussion about including `asteroid-2048` on all watches since it appears to be quite a popular little game. He agreed, I'll need to remove a buggy feature, then I'll probably add it to all watches :smile: